pkg/api/v1/conversion.go

```diff
@@ -294,6 +305,15 @@ func Convert_extensions_ReplicaSetStatus_to_v1_ReplicationControllerStatus(in *e
 	out.ReadyReplicas = in.ReadyReplicas
 	out.AvailableReplicas = in.AvailableReplicas
 	out.ObservedGeneration = in.ObservedGeneration
+	for _, cond := range in.Conditions {
```

Comment:
are there fuzzing tests for this round tripping? surprised nothing caught this before

Reply:
I added a manual test for round-tripping below (`TestReplicationControllerConversion`). I'll look into adding a fuzz test as well.

Reply:
definitely need a fuzz test to catch new fields